nav_msgs/Path trajectory
float64 duration
---
int32 result
int32 OK = 0
int32 ERROR = 1
